src/P/EitherT.hs

```diff
@@ -11,3 +13,6 @@ firstEitherT f = bimapEitherT f id
 
 secondEitherT :: (Functor m) => (a -> b) -> EitherT e m a -> EitherT e m b
 secondEitherT f = bimapEitherT id f
+
+eitherTFromMaybe :: Functor m => b -> m (Maybe a) -> EitherT b m a
+eitherTFromMaybe l r = EitherT $ maybe (Left l) pure <$> r
```

Comment:
The implementation looks a bit noisy /supernit

``` Haskell
eitherTFromMaybe :: Monad m => b -> m (Maybe a) -> EitherT b m a
eitherTFromMaybe l r = lift r >>= maybe (left l) return
```

But if the `Functor -> Monad` change isnt too palatable then nvm me